log/logger.go

```diff
@@ -71,13 +71,13 @@ func LevelAlignedString(l slog.Level) string {
 	case slog.LevelDebug:
 		return "DEBUG"
 	case slog.LevelInfo:
-		return "INFO "
```

Comment:
should we keep these spaces? I think they were put to keep the logs aligned

Reply:
yes but we don't do that for avalanche logs, I think it's better we align them?

Reply:
keeping them aligned is the upstream behavior as well so I suggest we keep it